<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DeviceListPageWidget</class>
 <widget class="QWidget" name="DeviceListPageWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="LineEdit" name="filterNameLineEdit" native="true">
     <property name="placeholderText" stdset="0">
      <string>Фильтрация по названию устройства...</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTableView" name="deviceListTableView">
     <property name="contextMenuPolicy">
      <enum>Qt::CustomContextMenu</enum>
     </property>
     <property name="editTriggers">
      <set>QAbstractItemView::EditKeyPressed</set>
     </property>
     <property name="dragDropOverwriteMode">
      <bool>false</bool>
     </property>
     <property name="alternatingRowColors">
      <bool>true</bool>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::SingleSelection</enum>
     </property>
     <property name="sortingEnabled">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="vlanInfoGroupBox">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="title">
      <string>Коммутатор xxx</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <property name="labelAlignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
      <property name="formAlignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
      </property>
      <item row="0" column="1">
       <widget class="QLabel" name="inetVlanValueLabel">
        <property name="text">
         <string>xxx</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLabel" name="iptvVlanValueLabel">
        <property name="text">
         <string>xxx</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="inetVlanDescLabel">
        <property name="layoutDirection">
         <enum>Qt::LeftToRight</enum>
        </property>
        <property name="text">
         <string>Internet Vlan:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="iptvVlanDescLabel">
        <property name="layoutDirection">
         <enum>Qt::LeftToRight</enum>
        </property>
        <property name="text">
         <string>IPTV Vlan:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="profileInfoGroupBox">
     <property name="title">
      <string>Олт xxx</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="serviceProfileOltLabel">
        <property name="text">
         <string>Сервисные профили</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLabel" name="multicastProfileOltLabel">
        <property name="text">
         <string>Мультикаст профили</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QListView" name="serviceProfileListView">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>100</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>200</height>
         </size>
        </property>
        <property name="editTriggers">
         <set>QAbstractItemView::NoEditTriggers</set>
        </property>
        <property name="alternatingRowColors">
         <bool>true</bool>
        </property>
        <property name="selectionBehavior">
         <enum>QAbstractItemView::SelectRows</enum>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QListView" name="multicastProfileListView">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>100</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>200</height>
         </size>
        </property>
        <property name="editTriggers">
         <set>QAbstractItemView::NoEditTriggers</set>
        </property>
        <property name="alternatingRowColors">
         <bool>true</bool>
        </property>
        <property name="selectionBehavior">
         <enum>QAbstractItemView::SelectRows</enum>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
  <action name="addDeviceAction">
   <property name="icon">
    <iconset>
     <normalon>:/images/add.png</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>Добавить</string>
   </property>
   <property name="toolTip">
    <string>Добавить</string>
   </property>
  </action>
  <action name="removeDeviceAction">
   <property name="icon">
    <iconset>
     <normalon>:/images/remove.png</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>Удалить</string>
   </property>
   <property name="toolTip">
    <string>Удалить</string>
   </property>
  </action>
  <action name="editDeviceAction">
   <property name="icon">
    <iconset>
     <normalon>:/images/edit.png</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>Редактировать</string>
   </property>
   <property name="toolTip">
    <string>Редактировать</string>
   </property>
  </action>
  <action name="selectDeviceAction">
   <property name="icon">
    <iconset>
     <normalon>:/images/open.png</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>Открыть</string>
   </property>
   <property name="toolTip">
    <string>Зайти на устройство</string>
   </property>
  </action>
  <action name="showVlanSwitchAction">
   <property name="text">
    <string>Вланы</string>
   </property>
   <property name="toolTip">
    <string>Информация о вланах коммутатора</string>
   </property>
  </action>
  <action name="editDslamBoardListAction">
   <property name="text">
    <string>Редактировать доски</string>
   </property>
   <property name="toolTip">
    <string>Редактировать доски</string>
   </property>
  </action>
  <action name="showProfilesOltAction">
   <property name="text">
    <string>Информация о профилях олт</string>
   </property>
   <property name="toolTip">
    <string>Информация о профилях олт</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>LineEdit</class>
   <extends>QWidget</extends>
   <header>lineedit.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
